### ISSUE TITLE
Applet doesn't "see" the application as installed

### ISSUE THREAD

**Issue by yuritoledo** [linuxmint/cinnamon-spices-applets#1889]
I'm on cinnamon 3.8.7, and have gpaste installed, via apt get.
I install the applet by applet menu and then, by the site, and I doesn't work.

**Comment by icarter09** [comment]
Cinnamon 3.8.8
Linux Mint 19  64bit
Intel Corp HD Graphics 530
Applet version 2.6  2018-04-15 11:04 UTC

@Feuerfuchs, I installed everything according to the README and experienced the same problem that @yuritoledo is having. I have restarted Cinnamon several times and tried reinstalling the applet as well, but had no luck. Keep seeing "[GPaste is not installed]" in the tray icon no matter what I do.
